channeldb/migration12/invoices.go

```diff
@@ -36,7 +36,22 @@ const (
 	acceptTimeType   tlv.Type = 9
 	resolveTimeType  tlv.Type = 11
 	expiryHeightType tlv.Type = 13
-	stateType        tlv.Type = 15
+	htlcStateType    tlv.Type = 15
```

Comment:
I don't think we want to use this migration where we write out all htlcs again to beautify these constant too?

Reply:
we definitely can, but it adds nothing functionally. i'd rather save this for the next migration where we do for htlcs since it is purely cosmetic and at this point can only introduce more bugs

Reply:
agrred